src/eShop.AppHost/eShop.AppHost.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
@davidfowl - did you mean to check this .sln file in? I don't see why it is necessary.

Reply:
I removed it.